recipes/hdbscan/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = "0.8" %}
+
+package:
+  name: hdbscan
+  version: {{ version }}
+
+source:
+  fn: hdbscan-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/h/hdbscan/hdbscan-{{ version }}.tar.gz
+  sha256: ea09e8ad60a1734e46000e71cc4b4ca274629e5ed9c54015cf4d4150c1238a17
+
+build:
+  number: 0
+  script: 
+    # FIXME: This is a hack to make sure the environment is activated.
+    # The reason this is required is due to the conda-build issue
+    # mentioned below.
+    #
+    # https://github.com/conda/conda-build/issues/910
+    #
+    - source activate "${CONDA_DEFAULT_ENV}"  # [unix]
+    - python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - toolchain
+    - python
+    - setuptools
+    - scikit-learn >=0.16
+    - cython >=0.17
+
+  run:
+    - python
+    - scikit-learn >=0.16
+
+test:
+  imports:
+    - hdbscan
+
+about:
+  home: http://github.com/lmcinnes/hdbscan
+  license: BSD 3-Clause
+  summary: 'Clustering based on density with variable density clusters'
+
+extra:
+  recipe-maintainers:
+    - LelandMcInnes
```

Comment:
Correction in the feedstock with PR ( https://github.com/conda-forge/hdbscan-feedstock/pull/1 ).